ruby-challenges.rb

```diff
@@ -11,6 +17,27 @@
 # Expected output: '221 is odd'
 
 
+def even_or_odd input_num 
+    if input_num % 2 == 0
+        p 'even'
+    elsif input_num % 2 == 1
+        p 'odd'
+    else 
+        p 'unknown'
+    end
+end
```

Comment:
Great!  I like that you added edge cases as well!  Ruby has some built-in methods that can help out with this as well.  Take a look at .even? and .odd? methods when you get the chance.